Servers/swagger.yaml

```
@@ -4336,6 +4338,254 @@ paths:
+              properties:
+                message:
+                  type: string
+                  example: internal server error
+                error:
+                  type: object
+    
+    post:
+      summary: Creates new training registar
+      tags: [training]
+      security:
+       - JWTAuth: []
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 7

(indentation)